services/api/src/resources/deploytargetconfig/resolvers.ts

```diff
@@ -262,44 +260,68 @@ export const deleteDeployTargetConfig: ResolverFn = async (
   return 'success';
 };
 
+interface UpdateDeployTargetConfigPatch {
+  branches?: string;
+  deployTarget?: number;
+  deployTargetProjectPattern?: string;
+  pullrequests?: string;
+  weight?: number;
+}
+
 export const updateDeployTargetConfig: ResolverFn = async (
   root,
-  { input },
-  { sqlClientPool, hasPermission, userActivityLogger }
+  { id, patch }: { id: number; patch: UpdateDeployTargetConfigPatch },
```

Comment:
updateDeployTargetConfig is broken for me when I use it locally, it complains with
```
Cannot destructure property 'branches' of 'patch' as it is undefined.
```

Patching locally with the following resolves the issue for me
```
  { input: {id, patch }}: {input: { id: number; patch: UpdateDeployTargetConfigPatch }},
```

Reply:
Thanks. My kingdom for auto-generated resolver types.